src/Cmi5.ts

```diff
@@ -74,10 +74,6 @@ export default class Cmi5 {
     Cmi5._instance = null;
   }
 
-  static get xapi(): XAPI | null {
```

Comment:
We're going to need to retain a method for accessing the internal xAPI instance so we don't lose functionality introduced in #85.

If we make it a regular getter function, unfortunately it will introduce a breaking change as we will no longer be able to access the xAPI instance on the class itself as it is static currently.

Reply:
Do you consider the `Cmi5._xapi` instance to be part of the committed public interface of the project? The leading `_` prefix is often used to indicate a "private" property, albeit only in an advisory capacity (at best).

If it is your intent to preserve that interface, then it seems like that should resolve to the xAPI client instance of the `Cmi5` singleton. In other words, I believe `Cmi5.instance.xapi === Cmi5._xapi` (should do, anyway).

Is that your intent?

Reply:
I restored the getter as a shortcut to the xAPI client on the `Cmi5` singleton. IOW, `Cmi5.xapi === Cmi5.instance.xapi`.